src/themes/broadband-deals/theme.json

```diff
@@ -870,6 +870,12 @@
           "lineHeight": "md"
         }
       },
+      "aspect-ratio-wrapper": {
+        "height": "0",
+        "paddingTop": "calc(189.67 / 336.98 * 100%)",
```

Comment:
```suggestion
        "paddingTop": "56.29%",
```

Reply:
They are image's width and height, calculating aspect-ratio. I will change them just to percentages. Thanks.